stablehlo/dialect/Base.cpp

```diff
@@ -146,5 +146,24 @@ LogicalResult verifyBounds(ArrayRef<int64_t> bounds, ShapedType type,
   return success();
 }
 
+ArrayRef<int64_t> encodingToBounds(Attribute encoding) {
+  if (!encoding) return {};
```

Comment:
You don't need this line. If encoding is null, then `dyn_cast_or_null` will not crash but return null, which is what we want.

Reply:
OK removed.